### PR TITLE
pkg/services/servicetest: add Run and RunHealthy helpers

### PR DESCRIPTION
**Pull request by jmank88**
Supports:
- https://github.com/smartcontractkit/chainlink/pull/11354

### CODE REVIEW

**Review by nolag — DISMISSED** [review]

File: pkg/services/servicetest/run.go [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+func Run(tb testing.TB, s Runnable) {
+	tb.Helper()
+	require.NoError(tb, s.Start(tests.Context(tb)), "service failed to start")
+	tb.Cleanup(func() { assert.NoError(tb, s.Close(), "error closing service") })
+}
+
+// RunHealthy fails tb if the service fails to start, close, is never ready, or is ever unhealthy (based on periodic checks).
+//   - after starting, readiness will always be checked at least once, before closing
+//   - if ever ready, then health will be checked at least once, before closing
+func RunHealthy(tb testing.TB, s services.Service) {
```
Comment (patrickhuie19) [inline pkg/services/servicetest/run.go:33]: Is there a sensible unit test for `RunHealthy`?

Comment (patrickhuie19) [reply pkg/services/servicetest/run.go:33]: Hmm. Probably not. The use in pkg/loop is likely enough. 

Related: should we add this file to sonar exceptions so we don't mess with coverage stats?

Comment (jmank88) [reply pkg/services/servicetest/run.go:33]: Added a test.

File: pkg/services/servicetest/run_test.go [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+	"github.com/stretchr/testify/assert"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/services"
+)
+
+func TestRunHealthy(t *testing.T) {
+	for _, test := range []struct {
+		name    string
+		s       services.Service
+		expFail bool
```
Comment (jmank88) [inline pkg/services/servicetest/run_test.go:18]: I tried to validate error messages too, but they contain whole stacks from testify internals.

**Review by patrickhuie19 — APPROVED** [review]

**Pull request merged**